From 706f6b313074b5c6ab015440313800d12f3c53e1 Mon Sep 17 00:00:00 2001
From: Changhuang Liang <changhuang.liang@starfivetech.com>
Date: Wed, 27 Sep 2023 06:07:34 -0700
Subject: [PATCH 10/12] pmdomain: starfive: Update prefixes for AON power
 domain

Use "JH7110_AON_PD_" prefix for AON power doamin for JH7110 SoC.

Reviewed-by: Walker Chen <walker.chen@starfivetech.com>
Signed-off-by: Changhuang Liang <changhuang.liang@starfivetech.com>
Link: https://lore.kernel.org/r/20230927130734.9921-3-changhuang.liang@starfivetech.com
Signed-off-by: Ulf Hansson <ulf.hansson@linaro.org>
---
 drivers/pmdomain/starfive/jh71xx-pmu.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/pmdomain/starfive/jh71xx-pmu.c b/drivers/pmdomain/starfive/jh71xx-pmu.c
index 2724bee7e..74720c09a 100644
--- a/drivers/pmdomain/starfive/jh71xx-pmu.c
+++ b/drivers/pmdomain/starfive/jh71xx-pmu.c
@@ -419,11 +419,11 @@ static const struct jh71xx_pmu_match_data jh7110_pmu = {
 };
 
 static const struct jh71xx_domain_info jh7110_aon_power_domains[] = {
-	[JH7110_PD_DPHY_TX] = {
+	[JH7110_AON_PD_DPHY_TX] = {
 		.name = "DPHY-TX",
 		.bit = 30,
 	},
-	[JH7110_PD_DPHY_RX] = {
+	[JH7110_AON_PD_DPHY_RX] = {
 		.name = "DPHY-RX",
 		.bit = 31,
 	},
-- 
2.40.1

